src/redux/redux-state.js

```diff
@@ -0,0 +1,32 @@
+import { createStore, combineReducers } from 'redux';
+
+import profileReducer from './profileReducer';
+import dialogsReducer from './dialogsReducer';
+import usersReducer from './usersReducer';
+
+
```

Comment:
Consider removing this redundant empty line.

Reply:
sure it's redundant line